rephrase optional step for datadog metrics (#1702)
astronomer/docs: astro/deployment-metrics.md

   - **Key:** `DATADOG_API_KEY`
   - **Value:** `<Your-Datadog-API-key>`.
   Select the **Secret?** checkbox. This ensures that your Datadog API key is saved securely and is not available to Workspace users in plain text.

> 4. Optional. If you use a Datadog site that is not the default (datadoghq.com), add the following environment variable:

Reviewer: ```suggestion
4. Optional. If your organization doesn't use the default Datadog site datadoghq.com, add the following environment variable:
```

> 4. Optional. If you don't want to use the default Datadog site (datadoghq.com) defined by Astronomer, add the following environment variable:

Reviewer 2: ```suggestion
4. Optional. If you use a Datadog site that is not the default (datadoghq.com), add the following environment variable:
```
